.npmrc

```diff
@@ -1 +1,2 @@
-save-exact=true
\ No newline at end of file
+save-exact=true
+node-linker=hoisted
```

Comment:
we should only use the hoisted node linker as an absolutely last resort after trying a bunch of other things

this makes pnpm less performant

Reply:
I will remove it and find other solutions first